xml/System.Collections.Generic/Stack`1+Enumerator.xml

```diff
@@ -72,6 +72,8 @@
   
  <xref:System.Collections.Generic.Stack%601.Enumerator.Current%2A> returns the same object until <xref:System.Collections.Generic.Stack%601.Enumerator.MoveNext%2A> is called. <xref:System.Collections.Generic.Stack%601.Enumerator.MoveNext%2A> sets <xref:System.Collections.Generic.Stack%601.Enumerator.Current%2A> to the next element.  
   
+ The order of enumeration of items proceeds from top to bottom (newest pushed to oldest pushed).  
```

Comment:
Are "top" and "bottom" well-defined/understood terms in the context of stacks? Also, "newest pushed" to "oldest pushed" is technically correct but it doesn't describe how the intermediate elements are enumerated. We might want to use terminology that is accepted in the literature for stacks.

Reply:
I believe top and bottom are commonly used and we use them elsewhere in the Stack documentation.

Reply:
```suggestion
Stack elements are enumerated from the top to the bottom of the stack.
```

Reply:
And move this to line 69 (add it to the existing paragraph about enumerators).